### DISCUSSION TITLE
Version 14.0 - in Recognition Mode Image for ANA2 und ANA3 are distorted

### DISCUSSION THREAD

**Discussion by ujagusch** [jomjol/AI-on-the-edge-device#1968] [General]
In Release: v14.0.0 (Commit: fa69272+) werden im Recognition Mode die beiden Bilder für ANA2 und ANA3 nicht korrekt dargestellt.

Die Werte werden korrekt angezeigt, nur die Images nicht.

![image](https://user-images.githubusercontent.com/61631767/216593834-54401b76-b3b7-4f02-b385-76245c3c8d8e.png)

UJagusch

**Comment by caco3** [comment]
hmm, on my system it is ok.

![SmartSelect_20230203_131039_Chrome](https://user-images.githubusercontent.com/1783586/216601021-daea869e-fce5-4f70-8390-dc723de86986.jpg)


maybe another config issue.

you could try the demo mode and see if your config has an issue there as well: https://jomjol.github.io/AI-on-the-edge-device-docs/Demo-Mode/

**Comment by ujagusch** [comment]
Dazu kommt noch das der analoge Readout  8 - 8 - 9 ist, dies wird auch auf der Overview Seite so angezeigt und auch sauber weiter hochgezählt.

Auf der Recognition Seite wird hier allerdings 9 - 8 - 9 angezeigt.

![image](https://user-images.githubusercontent.com/61631767/216611522-1dd70a72-17e1-4d77-98b1-afed84be0462.png)

Seltsam ist der Unterschied zwischen Overview und Recognition Seite - ausser es werden hier zwei unterschiedlichen, unabhängige Zeichenerkennungen durchgeführt, berechnet und angezeigt.

> hmm, on my system it is ok.

erinnert mich an die ersten Tipps bei Windows -" Bei mir läuft alles , installier mal alles neu" 

Wo sollte das Problem in der Config sein, wenn von 4 (3) Bildern 2 richtig angezeigt werden, zwei aber nicht?

UJagusch

**Comment by ujagusch** [comment]
ok - hier nochmal das Problem

![image](https://user-images.githubusercontent.com/61631767/216759070-3a000724-462d-4f5d-9589-2edf9051e663.png)

Die angezeigten Werte für RAW VALUE und VALUE sind korrekt
Wenn man aber auf der RECOGNITION SEITE nachschaut sieht es so aus:

![image](https://user-images.githubusercontent.com/61631767/216759800-786a0a5c-cb21-4b3a-8171-7e88ecfdc932.png)

Die Anzeige für ANA1 ist korrekt, aber falsch ausgelesen / interpretiert - statt 1 müsste es 0 Sein.
ANA2 und ANA3 werden korrekt ausgelesen, dafür werden die Images nicht angezeigt.

Ein weiterer Effekt - in Recognition und Overview wird das Übersichtsbild zuerst im "original" Modus angezeigt, so verdreht wie es aufgenommen wurde. Erst nach einer gewissen Zeit, nach erneutem Aufruf wird das Bild korrekt gedreht dargestellt.
Ich vermute dies hängt vom Zeitpunkt der Aufnahme ab und der anschliessenden Verarbeitung? Wenn dem so ist, vlt. könnt man einen Text einblenden "Processing" oder so ähnlich?

Das ändert aber noch nichts an dem nicht darstellen von ANA2 und ANA3.

UJagusch

**Comment by jomjol** [comment]
Also das Problem bei `x0.001` ist nachvollziehbar, da das ROI vermutlich ganz leicht über den Rand des Bildes geht und dann tritt das auf. Für das zweite `x0,01` kann ich es mir aktuell nicht erklären!
